libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -323,6 +375,134 @@ AP_GPS_SBF::process_message(void)
             state.status = AP_GPS::GPS_OK_FIX_2D;
         }
                     
+        return true;
+    }
+    case INSNavGeod:
+    {
+        const msg4226 &temp = sbf_msg.data.msg4226u;
+
+
+        // Update time state
+        if (temp.WNc != 65535) {
+            state.time_week = temp.WNc;
+            state.time_week_ms = (uint32_t)(temp.TOW);
+        }
+
+        check_new_itow(temp.TOW, sbf_msg.length);
+        state.last_gps_time_ms = AP_HAL::millis();
+
+        // Update velocity state (don't use −2·10^10)
+        if (temp.Vn > -200000) {
+            state.velocity.x = (float)(temp.Vn);
+            state.velocity.y = (float)(temp.Ve);
+            state.velocity.z = (float)(-temp.Vu);
+
+            state.have_vertical_velocity = true;
+
+            float ground_vector_sq = state.velocity[0] * state.velocity[0] + state.velocity[1] * state.velocity[1];
+            state.ground_speed = (float)safe_sqrt(ground_vector_sq);
+
+            state.ground_course = wrap_360(degrees(atan2f(state.velocity[1], state.velocity[0])));
+
+        }
+
+        // Update position state (don't use -2·10^10)
+        if (temp.Latitude > -200000) {
+            state.location.lat = (int32_t)(temp.Latitude * RAD_TO_DEG_DOUBLE * (double)1e7);
+            state.location.lng = (int32_t)(temp.Longitude * RAD_TO_DEG_DOUBLE * (double)1e7);
+            state.location.alt = (int32_t)(((float)temp.Height - temp.Undulation) * 1e2f);
+        }
+
+        // Update yaw (don't use -2·10^10)
+        if (temp.Heading > -200000) {
```

Comment:
Same thing as the heading variant, you want to grab the accuracy estimate information.

Reply:
I have look into the INSNAvGeod message and the only accuracy information is the HeadingStdDev (standard deviataion of heading). Its value is in degree. This is something else like the Cov_HeadHead valeu of the AttCovEuler message. This value is in degree². So which one is used as the gps_yaw_accuracy in the future? If using the AsteRx-i S we have to use the accuracy information in the INSNavGeod message. If using the AsteRx-m2a with dual antenna we have to use the AttEauler + the AttCovEuler messages. So the accuracy information is different. It a little bit confusing